src/commands/games/blackjack.ts

```diff
@@ -74,7 +61,7 @@ class BlackjackCommand extends BaseCommand {
     Returns true if the reaction corresponds to a valid action and is from the player
   */
   private reactFilter(reaction: MessageReaction, user: User, authorId: string): boolean {
-    return ['🇸', '🇭', '🇶'].includes(reaction.emoji.name) && user.id === authorId;
+    return reaction.emoji.name !== null && ['🇸', '🇭', '🇶'].includes(reaction.emoji.name) && user.id === authorId;
```

Comment:
which case does a message reaction cause a null name to occur?

Reply:
It just looks like with discord.js v13, it's possible for the emoji's name to be null, so I had to include that condition to prevent an error. https://discord.js.org/#/docs/discord.js/stable/class/Emoji?scrollTo=name